build-system/tasks/visual-diff/snippets/iframe-wrapper.js

```diff
@@ -2,8 +2,16 @@
 // with an <iframe>, so that we can perform viewport-constrained visual diff
 // tests.
 
-// The texts __WIDTH__ and __HEIGHT__ will be replaced at execution time with
-// the viewport's size from the visual test's config.
+// The following strings will be replaced at execution time:
+// * __WIDTH__ and __HEIGHT__ with the viewport's size from the visual test's config.
+// * __PERCY_CSS__ with Percy-specific CSS.
+
+if ('__PERCY_CSS__' !== '__' + 'PERCY_CSS' + '__') {
```

Comment:
I know you already merged this but... did you consider doing: `if (!(document.querySelector('style[data-percy-specific-css]'))` 🙃

Reply:
1. It's `link[data-percy-specific-css]` as Percy includes it 😉
2. Unfortunately, the option is evaluated afterwards when taking the `percySnapshot()`, so it'd be too early to read it from the top-level document.

Reply:
Got it. Carry on 👑